app/views/play/level/modal/HeroVictoryModal.coffee

```diff
@@ -416,7 +416,8 @@ module.exports = class HeroVictoryModal extends ModalView
     campaign = @level.get 'campaign'
     if @level.get('slug') in campaignEndLevels
       campaign = ''  # Return to campaign selector
-    if (campaign is 'dungeon' or @level.get('slug') in ['kithgard-gates', 'game-grove']) and storage.load('should-return-to-game-dev-hoc')
+    gdHocLevels = ['kithgard-gates', 'over-the-garden-wall', 'vorpal-mouse', 'forest-incursion', 'them-bones', 'behavior-driven-development', 'seeing-is-believing', 'persistence-pays', 'game-grove']
```

Comment:
Are these level names locked down in practice?  Should use ids if not.

Reply:
Yes, they're as locked down as the rest of the levels we do this for. It's expected, I believe, that if we change names of published levels that the codebase needs to be checked for hardcodings like this. Would be good to figure out a better system, but that's a larger project.